code/modules/background/origins/origins/human/coalition.dm

```diff
@@ -13,12 +13,12 @@
 		/singleton/origin_item/origin/other_coalition
 	)
 
-/singleton/origin_item/origin/xanu_free_league
-	name = "Xanu Free League"
-	desc = "The Xanu Free League, and its capital of Xanu Prime, have long been considered some of the most important figures in the Coalition of Colonies. Xanusanii are a diverse body of people that value their freedom highly, and are known throughout the broader Spur as traders. Much of Xanu's surface remains uninhabitable due to damage inflicted upon the planet by the Solarian Alliance during the Interstellar War."
-	possible_accents = list(ACCENT_XANU, ACCENT_COC)
+/singleton/origin_item/origin/xanu_prime
+	name = "Xanu Prime"
+	desc = "Xanu Prime has long been considered one of the most important planets in the Coalition of Colonies, housing its current capital. Xanans are a diverse body of people that value their freedom highly, and are known throughout the broader Spur as traders. Much of Xanu's surface is scarred due to damage inflicted upon the planet by the Solarian Alliance during the Interstellar War."
+	possible_accents = list(ACCENT_XANU)
 	possible_citizenships = CITIZENSHIPS_COALITION
-	possible_religions = RELIGIONS_COALITION
+	possible_religions = list(RELIGION_NONE, RELIGION_CHRISTIANITY, RELIGION_ISLAM, RELIGION_BUDDHISM, RELIGION_SHINTO, RELIGION_HINDU, RELIGION_TAOISM, RELIGION_JUDAISM, RELIGION_SIKHISM, RELIGION_OTHER, RELIGION_TRINARY, RELIGION_STOLITISM, RELIGION_MOROZ, RELIGION_LUCEISM)
```

Comment:
why?

Reply:
To add Stolitism, Tribunalism, and Luceism - that was the original whole point of this PR. The minorities in Xanu are told to take a Xanan origin, but the lore specifically states they still follow their old religions. And so, I added them here.

Reply:
i mean why a list instead of a `RELIGIONS_XANU_PRIME` define.

Reply:
Every origin with a unique list rather than a shared, common one does that - Imperial Frontier, Coalition Spacers, Assunzione, and so on.

Reply:
👍